test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -90,6 +92,29 @@ export class WebElementWrapper {
     return otherWebElements.map(e => this._wrap(e));
   }
 
+  private retryCount: number = 3;
```

Comment:
Let's make this and the `errors` array within `retryCall()` constants at the top of the file.